showcase/src/app/demo-ui/pages/dropdown/demo-dropdown-page.component.html

```diff
@@ -6,23 +6,29 @@ <h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
 		filesPath="dropdown/reactive-form-dropdown"
 		[exampleTitle]="'SHOWCASE.DEMO.DROPDOWN.REACTIVE_FORM' | translate"
 	>
-		<stark-dropdown
-			dropdownId="reactiveFormDropdown"
-			[options]="serviceDropdownOptions"
-			[dropdownFormControl]="serviceFormControl"
-			placeholder="SHOWCASE.DEMO.DROPDOWN.SELECT_SERVICE"
-			optionIdProperty="id"
-			optionLabelProperty="value"
-			required
-		>
-		</stark-dropdown>
-
+		<mat-form-field>
+			<stark-dropdown
+				dropdownId="reactiveFormDropdown"
+				[options]="serviceDropdownOptions"
+				[formControl]="serviceFormControl"
```

Comment:
Shouldn't the `serviceFormControl` have also the `required` validator set? Currently it does not... which is confusing